apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/useSiteOptions.js

```diff
@@ -114,5 +102,13 @@ export default function useSiteOptions(
 		} );
 	}
 
-	return { siteOptions, setSiteOptions, onSave };
+	function handleChange( value ) {
+		// The following is a temporary fix. Setting this fake attribute is used to flag
+		// the content as dirty to editor and enable Update/Publish button. This should be
+		// removed once updating of site options handled in core editor
+		setAttributes( { updated: Date.now() } );
```

Comment:
It is interesting that this works to flag the editor state as dirty, even though the fse blocks do not have any attributes set and so this doesn't actually update any attribute data in the block content.

Reply:
I think it's ok to use this hack while we confirm if the Gutenberg behavior here is deliberate or not (paAmJe-tF-p2 #comment-1329).

Reply:
Wacky.